src/Controller/Web/Admin/SpaceTypeAdminController.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Controller\Web\Admin;
+
+use App\Service\Interface\SpaceTypeServiceInterface;
+use Symfony\Component\HttpFoundation\Response;
+
+class SpaceTypeAdminController extends AbstractAdminController
+{
+    private const string LIST = 'space-type/list.html.twig';
+
+    public function __construct(
+        private readonly SpaceTypeServiceInterface $service,
+    ) {
+    }
+
+    public function list(): Response
+    {
+        $spaceTypes = $this->service->list();
+
+        return $this->render(self::LIST, [
+            'spaceTypes' => $spaceTypes,
+        ]);
```

Comment:
```suggestion
        return $this->render('space-type/list.html.twig', [
            'spaceTypes' => $spaceTypes,
        ]);
```

Sugiro que a constante LIST seja removida, pois é utilizada apenas uma vez, tornando sua existência desnecessária. Manter a string diretamente no método render melhora a legibilidade do código e elimina uma referência indireta. Além disso, isso pode evitar um pequeno overhead ao resolver a constante em tempo de execução.